oddOrEven.js

```diff
@@ -0,0 +1,31 @@
+/*Task:
+Given a list of integers, determine whether the sum of its elements is odd or even.
+
+Give your answer as a string matching "odd" or "even".
+
+If the input array is empty consider it as: [0] (array with a zero).
+
+Examples:
+Input: [0]
+Output: "even"
+
+Input: [0, 1, 4]
+Output: "odd"
+
+Input: [0, -1, -5]
+Output: "even"
+Have fun!
+*/
+function oddOrEven(array) {
+   const sum = array.reduce((accumulator, value) => {
+     return accumulator + value;
+   }, 0)
+   if(array === null || array === 0) {
```

Comment:
Hmm ustawiamy tutaj array, ale nigdzie go już nie używamy (?)
Poza tym powinniśmy generalnie unikać nadpisywania/modyfikowania zmiennych które są argumentami funkcji, tj które przyszły z zewnątrz

Reply:
Racja, zmylił mnie troche ten warunek "If the input array is empty consider it as: [0] (array with a zero)." i niepoitrzebnie to dodałem - bez tego też normalnie przeszedł challenge. Usuwam